diff --git a/dist/esm/index.js b/dist/esm/index.js
index e3c478107c859d86196b8e38bb4431dc1ec91f8d..4d9d7751c38651360971679e9068e0dc3b37f199 100644
--- a/dist/esm/index.js
+++ b/dist/esm/index.js
@@ -1,5 +1,3 @@
-import NodeWallet from "./nodewallet";
-import { isBrowser } from "./utils/common.js";
 export { default as BN } from "bn.js";
 export * as web3 from "@solana/web3.js";
 export { getProvider, setProvider, AnchorProvider, } from "./provider.js";
@@ -9,8 +7,6 @@ export * from "./coder/index.js";
 export * as utils from "./utils/index.js";
 export * from "./program/index.js";
 export * from "./native/index.js";
-if (!isBrowser) {
-    exports.workspace = require("./workspace.js").default;
-    exports.Wallet = require("./nodewallet.js").default;
-}
+export { default as workspace } from "./workspace.js";
+export { default as Wallet } from "./nodewallet.js";
 //# sourceMappingURL=index.js.map
